docs/tutorial/offscreen-rendering.md

```diff
@@ -1,59 +1,68 @@
 # Offscreen Rendering
 
-Offscreen rendering lets you obtain the content of a browser window in a bitmap,
-so it can be rendered anywhere, for example on a texture in a 3D scene. The
-offscreen rendering in Electron uses a similar approach than the [Chromium
-Embedded Framework](https://bitbucket.org/chromiumembedded/cef) project.
+## Overview
 
-Two modes of rendering can be used and only the dirty area is passed in the
-`'paint'` event to be more efficient. The rendering can be stopped, continued
-and the frame rate can be set. The specified frame rate is a top limit value,
-when there is nothing happening on a webpage, no frames are generated. The
-maximum frame rate is 60, because above that there is no benefit, only
-performance loss.
+Offscreen rendering lets you obtain the content of a `BrowserWindow` in a
+bitmap, so it can be rendered anywhere, for example, on texture in a 3D scene.
+The offscreen rendering in Electron uses a similar approach to that of the
+[Chromium Embedded Framework](https://bitbucket.org/chromiumembedded/cef)
+project.
 
-**Note:** An offscreen window is always created as a [Frameless Window](../api/frameless-window.md).
+*Notes*:
 
-## Rendering Modes
+* There are two rendering modes that can be used (see the section below) and only
+the dirty area is passed to the `paint` event to be more efficient. You can
+stop/continue the rendering as well as set the frame rate.
 
-### GPU accelerated
+  The specified frame rate is a top limit value, when nothing happening on a
+webpage, no frames are generated. The maximum frame rate is 60 because greater
+values bring only performance losses with no benefits.
```

Comment:
```suggestion
  * The maximum frame rate is 60 because greater values bring only performance losses with no benefits.
  * When nothing is happening on a webpage, no frames are generated. 
```